packages/syntax-icons/package.json

```diff
@@ -4,18 +4,18 @@
   "version": "4.7.0",
   "type": "module",
   "main": "./dist/index.js",
-  "module": "./dist/index.mjs",
   "types": "./dist/index.d.ts",
   "sideEffects": false,
   "license": "Apache-2.0",
   "files": [
     "dist/**"
   ],
   "scripts": {
-    "build": "tsup src/icons/*.tsx --format esm,cjs --dts --external react",
-    "dev": "tsup src/icons/*.tsx --format esm,cjs --watch --dts --external react",
+    "build": "tsup src/icons/*.tsx --format esm --dts --external react",
+    "check-exports": "attw $(pnpm pack) --ignore-rules=cjs-resolves-to-esm --ignore-rules=no-resolution",
+    "dev": "tsup src/icons/*.tsx --format esm --watch --dts --external react",
     "lint": "TIMING=1 eslint \"src/**/*.ts*\" --max-warnings 0",
-    "clean": "rm -rf .turbo && rm -rf node_modules && rm -rf dist",
+    "clean": "rm -rf .turbo && rm -rf node_modules && rm -rf dist && rm -rf *.tgz",
```

Comment:
We're we exporting `tgz` when we didn't need to?

Reply:
Going off the name does `check-exports` generate a tarball to check if the exports are esm-only then now we have to clean up the tarball? Why is this necessary?

Reply:
Oh wait. `attw` is are the types wrong. It's generating the tarball then checking if the types are correct on it then deleting it. This makes sense but how is this different then typechecking? Is it more thorough? Or maybe its entirely different?

Reply:
@williamnewman94 see https://github.com/Cambly/syntax/pull/544/files#diff-3d7766fca98eabc07616b81b6551c3f305814a858011e90ff75263bb72d79159R15 - the `pack` command creates a tarball (`.tgz` file), so we do want to make sure it never gets committed to this repo & yes, that we do clean it up